internal/controller/csiaddons/networkfence_controller.go

```diff
@@ -59,6 +59,16 @@ func validateNetworkFenceSpec(nwFence *csiaddonsv1alpha1.NetworkFence) error {
 	if nwFence == nil {
 		return errors.New("NetworkFence resource is empty")
 	}
+
+	if nwFence.Spec.NetworkFenceClassName != "" {
+		if nwFence.Spec.Cidrs == nil {
+			return errors.New("required parameter cidrs is not specified")
```

Comment:
return errors.New("required Spec.Cidrs is not specified")

Reply:
Done!